_data/soulhunter10737/soulhunter10737.md

```diff
@@ -0,0 +1,7 @@
+---
+name: SHIBAM DHAR
+institution: KCCITM
+profile_pic: soulhunter10737.jpj
```

Comment:
Incorrect file name, please do one of the following:
1. Change `soulhunter10737.jpj` to `soulhunter10737.jpg` in the markdown file.
2. Rename `soulhunter10737.jpg` to `soulhunter10737.jpj`.
I'd suggest you make the change in the markdown file!

Reply:
Thank You so much for mentioning this. Yes I have made the required changes.

Reply:
@Mohammed-Shoaib Do let me know for further updates.

Reply:
It doesn't seem like you have made the required changes. Did you forget to push the changes? :thinking:

Reply:
Yes I did,  I issued it again now. https://github.com/education/graduation/pull/4376 I hope this will do :/

Reply:
Do let me know @Mohammed-Shoaib

Reply:
Oh okay, you could have actually just done the changes in this PR instead of making a new pull request :grinning:
Kindly close this PR then since it's not required anymore.